clients/pkg/promtail/targets/gcplog/push_translation.go

```diff
@@ -73,45 +74,46 @@ func translate(m PushMessage, other model.LabelSet, useIncomingTimestamp bool, r
 		labels[model.LabelName(lbl.Name)] = model.LabelValue(lbl.Value)
 	}
 
-	// add labels coming from scrapeconfig
+	// Add fixed labels coming from the target configuration
 	labels = labels.Merge(other)
 
-	ts := time.Now()
-
 	decodedData, err := base64.StdEncoding.DecodeString(m.Message.Data)
 	if err != nil {
 		return api.Entry{}, fmt.Errorf("failed to decode data: %w", err)
 	}
-	line := string(decodedData)
-
-	if useIncomingTimestamp {
```

Comment:
Where did this go?

Reply:
`parseGCPLogsEntry` has a similar logic that uses fields that that make more sense to use as timestamp, instead of the push one.